ui/imports/shared/status/StatusStickersPopup.qml

```diff
@@ -256,10 +256,11 @@ Popup {
             StatusScrollView {
                 id: installedStickersSV
                 height: 40
+                Layout.fillWidth: true
+                ScrollBar.vertical.policy: ScrollBar.AlwaysOff
 
                 RowLayout {
                     id: stickersRowLayout
-                    width: installedStickersSV.availableWidth
                     spacing: Style.current.padding
```

Comment:
could you please check with a `Tracer` that the `RowLayout` here has its dimensions set properly?